zeq/publisher.cpp

```diff
@@ -94,13 +95,32 @@ class Publisher
     }
 
     const std::string& getAddress() const { return _address; }
+    const lunchbox::URI uri;
 
 private:
-    void _initService( std::string host, uint16_t port )
+    void _initService( const uint32_t announceMode, std::string host,
+                       uint16_t port )
     {
         _initAddress( host, port );
-        if( lunchbox::Servus::isAvailable( ))
-            _service.announce( port, _address );
+        if( announceMode == ANNOUNCE_NONE || !(announceMode&ANNOUNCE_ZEROCONF) )
```

Comment:
check for NONE bit? If mode bits are set to ZEROCONF|NONE (as a result of automated code), what should happen?